dyno-core/src/main/java/com/netflix/dyno/connectionpool/impl/lb/HostSelectionWithFallback.java

```diff
@@ -207,7 +210,7 @@ private boolean attemptFallback() {
 			try {
 				
 				HostConnectionPool<CL> fallbackHostPool = 
-						(op != null) ? remoteDCSelector.getPoolForOperation(op) : remoteDCSelector.getPoolForToken(token);
+						(op != null) ? remoteDCSelector.getPoolForOperation(op,hashtag) : remoteDCSelector.getPoolForToken(token);
```

Comment:
not related to this PR but is this really a "remote DC selector"?

Reply:
I think this is a wrong name. Looking in the code `String dc = host.getRack();`... and then `HostSelectionStrategy<CL> remoteSelector = remoteDCSelectors.get(dc);`. I can fix this.

Reply:
Refactored the names. They will appear in the upcoming commit.